include/SFML/Graphics/Font.hpp

```diff
@@ -250,8 +251,25 @@ class SFML_GRAPHICS_API Font
     ///
     /// \return Texture containing the glyphs of the requested size
     ///
+    /// \deprecated There may now be multiple textures instead of one.
+    /// Use getGlyph(...).texture instead.
+    ///
+    /// \see getGlyph
+    ///
+    ////////////////////////////////////////////////////////////
+    SFML_DEPRECATED const Texture& getTexture(unsigned int characterSize) const;
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Retrieve the texture id
+    ///
+    /// The texture id is used internally by sf::Text.
+    ///
+    /// \param characterSize Reference character size
+    ///
+    /// \return Texture id containing glyphs of the requested size
+    ///
     ////////////////////////////////////////////////////////////
-    const Texture& getTexture(unsigned int characterSize) const;
+    Uint64 getTextureId(unsigned int characterSize) const;
```

Comment:
Even though `getTexture` was mostly for internal usage, am I correctly understanding that with this change you can't retrieve the textures as a user anymore?

Reply:
The user can access the texture with `getGlyph().texture` just like `Text` does - it's written in the documentation of the deprecated `getTexture()`.
`getTextureId()` is used internally only for invalidating the geometry, not for building it.